### ISSUE TITLE
Histogram not working properly, according to example

### ISSUE THREAD

**Issue by lobis** [mipt-npm/plotly.kt#70]
I believe there is an issue with the histogram examples (or perhaps with histograms itself). I have created a simple js histogram rendering using https://github.com/mipt-npm/plotly.kt/blob/master/examples/src/main/kotlin/histogram/BasicHistogram.kt as a reference. The code is the following: (version `0.3.1`)

```
    document.addEventListener("DOMContentLoaded", {
        val element = document.getElementById("test") as? HTMLElement
            ?: error("Element with id 'test' not found on page")

        console.log("element loaded")

        val rnd = Random
        val values = List(500) { rnd.nextDouble() }

        element.plot {

            histogram {
                x.numbers = values
                name = "Random data"
            }

            layout {
                bargap = 0.1
                title {
                    text = "Basic Histogram"
                    font {
                        size = 20
                        color("black")
                    }
                }
                xaxis {
                    title {
                        text = "Value"
                        font {
                            size = 16
                        }
                    }
                }
                yaxis {
                    title {
                        text = "Count"
                        font {
                            size = 16
                        }
                    }
                }
            }
        }
    })
```

I believe this should produce a good looking histogram however what is produced is the following:

![image](https://user-images.githubusercontent.com/35803280/112261180-a2768e00-8c6b-11eb-82ff-9ab23883873c.png)

Also there is another issue in the examples, the usage of `Random`, which according to the documentation (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.random/-random/) should be called like `Random.nextDouble()` instead of `Random().nextDouble()` as it is written in the examples.

**Comment by altavir** [comment]
Thanks, I've reproduced it on JS (works perfectly fine on the JVM). For some reason, it ignores the trace type parameter when used directly from JS.

The Random is not the issue. `Random()` function creates a new generator with a random seed whereas `Random` singleton (companion object) represents the default generator that is created on start. In this context, there is no difference.

**Comment by lobis** [comment]
> Thanks, I've reproduced it on JS (works perfectly fine on the JVM). For some reason, it ignores the trace type parameter when used directly from JS.
> 
> The Random is not the issue. `Random()` function creates a new generator with a random seed whereas `Random` singleton (companion object) represents the default generator that is created on start. In this context, there is no difference.

For some reason I couldn't get the `Random()` to work on my plot, I had to change it to `Random` as you see on the code in order to work. Probably some issue with dependencies on my side. (I imported `import kotlin.random.Random` to access `Random`).

**Comment by altavir** [comment]
Just for educational purposes. The problem is that I used the direct dynamic conversion of Kotlon structures to Json and the enum class is converted into structure instead of value. Like this:
![image](https://user-images.githubusercontent.com/9514268/112266658-5df7ec00-8c85-11eb-987f-68101f4494a8.png)
I will fix that shortly.

**Comment by altavir** [comment]
Fixed and will be available in 0.4.0

**Comment by lobis** [comment]
> Fixed and will be available in 0.4.0

Is the fix available in the currently published development version? `0.3.1-dev6` (if exists) when are those released? Thanks.

**Comment by altavir** [comment]
No, we need to migrate to a new version because there are some breaking changes and the package version change (kscience->space.kscience). I've published the `0.4.0-dev-2` to the dev repository. So you can use right now via 

```kotlin
reposirories{
  maven("https://repo.kotlin.link")
}

depdendencies{
  implementation("space.kscience:plotlykt-core:0.4.0-dev-2")
}
```
There are breaking changes in package names, so be sure to check examples in dev branch.
